src/stores/printing.js

```diff
@@ -90,5 +98,20 @@ export const usePrintingStore = defineStore('printing', {
       }
       return { success, data, errors }
     },
+
+    /**
+     * fetches the list of workflows from traction
+     * @returns {Object}  {success: Boolean, data: Object, errors: Array} Is the fetch successful? The data and any errors
+     */
+    async fetchWorkflows() {
```

Comment:
This probably needs a bit of work. The standard is to return the workflows with the steps and store them in 2 separate pieces of state. You can then bring them together in the component. I can go through this with you.